src/coreclr/jit/lower.cpp

```diff
@@ -8026,8 +8154,14 @@ void Lowering::LowerStoreIndirCoalescing(GenTreeStoreInd* ind)
 
         // Get coalescing data for the previous STOREIND
         GenTreeStoreInd* prevInd = prevTree->AsStoreInd();
-        if (!GetStoreCoalescingData(comp, prevInd->AsStoreInd(), &prevData))
+        if (!GetLoadStoreCoalescingData(comp, prevInd->AsStoreInd(), &prevData))
+        {
+            return;
+        }
+
+        if (!CanBeCoalesced(prevData, currData))
         {
+            CanBeCoalesced(prevData, currData);
```

Comment:
Is this second call necessary?

Reply:
Thanks for noticing it, yeah I was debugging a CQ issue :-) forgot to remove